invoice/src/main/java/org/killbill/billing/invoice/api/InvoiceApiHelper.java

```diff
@@ -86,14 +86,16 @@ public InvoiceApiHelper(final InvoicePluginDispatcher invoicePluginDispatcher, f
                                                                     final boolean isDryRun,
                                                                     final WithAccountLock withAccountLock,
                                                                     final LinkedList<PluginProperty> properties,
-                                                                    final CallContext context) throws InvoiceApiException {
+                                                                    final CallContext contextMaybeWithoutAccountId) throws InvoiceApiException {
```

Comment:
We should merge master back into `account-bcd-update` first.

Reply:
I thought that's what I did [here](https://github.com/killbill/killbill/pull/1484/commits/c07f7a9136d306f8a9584aa1fa583d764d0522e4) (last commit from the open PR)